templates/welcome_email.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+</head>
+
+<body>
+    <h3>Hi {{firstName}},</h3>
+    <p>Thanks for filling out the form on <a href="https://www.rememberthee.com" target="_blank">rememberthee.com</a>.
+        Remember Thee is available for download on iOS only. <br>You will receive an email from Apple's app testing
+        product, <a href="https://testflight.apple.com" target="_blank">TestFlight</a>, with instructions on how to
+        download Remember Thee shortly.</p>
+
+    <p>A few things to keep in mind while downloading and using Remember Thee:</p>
+    <ul>
+        <li>This version of Remember Thee does not contain all the features you saw on the website.
+            <br>They will be added in the coming days and weeks. If there was a feature you were particularly excited
+            about, please let me know.
+        </li><br>
+        <li>In general, if there are things you wish the app could do, or if it's not really working for you, please let
+            me know.
+        </li><br>
+        <li>If the app crashes, something looks visibly broken or doesn't work or is difficult to use, or you're having
+            trouble downloading the app. <br>
+            You can reach out to me either at this email or at help@rememberthee.com, and I'll get back to you as soon
+            as I can to help fix the problem.
+        </li><br>
+        <li>If you're enjoying the app, please feel free to share the link to the website with others. <br>
+            Anyone that signs up on the website will pretty much be given access to the beta version of the app.
+        </li>
+        <br>
+    </ul>
+    <p>Thanks, and let me know if you have any further questions, comments or problems. I hope Remember Thee will be
+        blessing to you either now, or in the future.</p>
+
+    <h5>Trevor Dowds, <br>Founder and Creator</h5>
+    <br>
+    <img src="/static/images/Logo.png">
```

Comment:
When I tested this out, this failed (see screenshot below). You probably need to specify the full path in the `<img>` element - `src="https://www.rememberthee.com/static/images/Logo.png"`

<img width="269" alt="Screen Shot 2021-11-18 at 9 47 23 AM" src="https://user-images.githubusercontent.com/5726560/142469044-404f666b-07b3-451a-be04-209dda6e01bd.png">